theano/tensor/tests/test_opt.py

```diff
@@ -3502,6 +3502,8 @@ def test_local_elemwise_sub_zeros():
     f = function([scalar], scalar - scalar, mode=mode)
     # Check optimized graph is correct
     assert isinstance(f.maker.fgraph.toposort()[0].op, T.Elemwise)
+    assert f.maker.fgraph.toposort()[0].op.name\
+        == 'Elemwise{second,no_inplace}'
```

Comment:
Good point. I guess in general it's not a good idea to compare to a string name of an op... I've fixed it now.